sonic-chassisd/scripts/chassisd

```diff
@@ -266,6 +301,16 @@ class ModuleUpdater(logger.Logger):
                                                             (CHASSIS_ASIC_ID_IN_MODULE_FIELD, str(asic_id))])
                     self.asic_table.set(asic_key, asic_fvs)
 
+        # In line card push the hostname of the module and num_asics to the chassis state db.
+        # The hostname is used as key to access chassis app db entries 
+        if not self._is_supervisor():
+           hostname_key = "{}{}".format(ModuleBase.MODULE_TYPE_LINE, int(self.my_slot) - 1)
+           hostname = try_get(device_info.get_hostname, default="None")
+           hostname_fvs = swsscommon.FieldValuePairs([(CHASSIS_MODULE_INFO_SLOT_FIELD, self.my_slot), 
```

Comment:
This line is causing crash on LC
https://github.com/sonic-net/sonic-buildimage/issues/16465

Looks like the code assumes the get_my_slot API to provide a string. Is that expected, I do not see it enforced by the module.